0010-connector-to-connector-protocol/0010-connector-to-connector-protocol.md

```diff
@@ -1,14 +1,72 @@
 ---
 title: The Connector to Connector protocol
-draft: 2
+draft: 3
 ---
 # Connector to Connector protocol
 
 ## Discovery
 
 Connectors discover their peers through out-of-band communication, or by looking at https://connector.land and contacting the administrator of another connector.
 
-Once peered, two connectors have a ledger between them; this is often a ledger with just two accounts, often administered collaboratively by the two connectors.
+Once peered, two connectors have a ledger between them; this is often a ledger with just two accounts.
+
+There are two ways for connectors to peer with each other: symmetric, and asymmetric. In symmetric peering, the ledger between the two connectors is administered
+collaboratively by the two connectors, optionally relying on a trusted third party (validator). In asymmetric peering, the ledger is administered by one of the two
+peers, and the other peer only keeps a non-authoritative shadow ledger.
+
+Regardless of whether the connectors peer symmetrically or asymmetrically, the initiating connector somehow needs to exchange information with the other connector.
+WebFinger can help with that.
+
+Whether discovery is done with or without use of WebFinger, each peer ends up knowing:
+
+* the protocol to use to send information like route broadcasts to the other connector
+* the protocol to use to add updates to the peering ledger
+* the currency code for the peering ledger
+* the currency scale for the peering ledger
+
+### WebFinger-based discovery
+In WebFinger-based discovery, both peers still need some out-of-band communication channel, over which one
```

Comment:
Maybe mention that this is normally something like talking in person or over email/chat